### DISCUSSION TITLE
Version Control in Data Science Projects

### DISCUSSION THREAD

**Discussion by hamzamsalah** [community/community#135887] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone! I'm curious to know how you manage version control in your data science projects, especially when handling large datasets and Jupyter notebooks.

**Comment by romhenri** [comment, marked as answer]
For version control in data science projects, you can use Git to manage your code and Jupyter notebooks, ensuring that all changes are tracked. Since large datasets can bloat the repository, keep them outside the Git repo. Instead, use DVC (Data Version Control) to track and version datasets without adding them directly to the repository, is a special case.

**Reply by hamzamsalah** [reply]
thank you for elaborating on that. I would assume large dataset don't get pushed.
